site/en/education/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Education apps on Chrome OS
+metadesc: Developing apps for education on Chrome OS.
+hero:
+  image: ix://landings/adapting-android.svg
+  alt: Illustration of an Android phone and Chrome OS device showing the same application running on both.
+date: 2022-03-21
+weight: -8
+---
+
+Chromebooks are the number one device in K-12 education globally with 50 million students and educators using them worldwide to create, collaborate and communicate[^1]. Chrome OS brings this exciting opportunity to developers by offering a unique environment that joins the power of the Web and Android on a single desktop.
+
+If you’re building for education, you can develop [Progressive Web Apps](https://chromeos.dev/en/web) (PWA) on Chrome OS and deliver advanced user experiences with [ever-expanding web APIs](https://chromeos.dev/en/web/powerful-pwas). Chrome OS also allows you to extend browser capabilities even further using the powerful extensions APIs available to [Chrome extensions](https://developer.chrome.com/extensions). Alternatively, you can [optimize your existing Android app](https://chromeos.dev/en/android) for Chromebooks with large screens. Because Web and Android are at the core of Chrome OS, PWAs and Android apps can be installed, pinned to the shelf, and found in the launcher, making them available at your user’s fingertips.
```

Comment:
```suggestion
If you’re building for education, you can develop [Progressive Web Apps](https://chromeos.dev/en/web) (PWA) on Chrome OS and deliver advanced user experiences with an [ever-expanding number of web APIs](https://chromeos.dev/en/web/powerful-pwas). Chrome OS also allows you to extend browser capabilities even further using the powerful APIs available to [Chrome extensions](https://developer.chrome.com/extensions). Alternatively, you can [optimize your existing Android app](https://chromeos.dev/en/android) for Chromebooks with large screens. Because Web and Android are at the core of Chrome OS, PWAs and Android apps can be installed, pinned to the shelf, and found in the launcher, making them available at your user’s fingertips.
```